cmd/swarm/config.go

```diff
@@ -101,6 +102,8 @@ func buildConfig(ctx *cli.Context) (config *bzzapi.Config, err error) {
 	config = envVarsOverride(config)
 	//override settings provided by command line
 	config = cmdLineOverride(config, ctx)
+	//validate configuration parameters
+	err = validateConfig(config)
```

Comment:
This shadows the previous assignment of `err` which is now not being checked (we should probably avoid named return arguments unless they are really needed)

Reply:
Fixed by returning right after the previous error assignment if it is not nil.